scripts/image_build.sh

```diff
@@ -56,7 +56,13 @@ if [ "$CD_REF_NAME" != "master" ] && [[ "$CD_REF_NAME" != *"release"* ]] && ( [
 fi
 
 # Include sqlite for production
-sed -i "/^group :production do/a\ \ gem 'sqlite3', '~> 1.3.6'" Gemfile
+sqliteCount="$(grep "gem 'sqlite3'" Gemfile | wc -l)"
+
+if [ $sqliteCount -lt 2 ]; then
+  echo "test"
```

Comment:
Forgot to remove some debugging code

Reply:
No that's necessary otherwise each time you run the build script you add the gem. It should only be added once.